site/_posts/2022-08-17-10log-fusionauth.md

```diff
@@ -9,7 +9,7 @@ tags: topic-community-story
 excerpt_separator: "<!--more-->"
 ---
 
-Kőműves Ákos a FusionAuth community member and team member at 10log. He chatted with us over email about how he and the 10log team are using FusionAuth to meet their auth needs. 
+Ákos Ákos is a FusionAuth community member and team member at 10log. He chatted with us over email about how he and the 10log team are using FusionAuth to meet their auth needs. 
```

Comment:
@mooreds this says his first name twice now.

Reply:
Ack! Thanks for catching that.

Reply:
Should be fixed in https://github.com/FusionAuth/fusionauth-site/commit/31bff727e622662242b04b94f77f73819eb8c11b